### DISCUSSION TITLE
Is there any way to call rofi with multiple rows being selected?

### DISCUSSION THREAD

**Discussion by Alexey104** [davatorium/rofi#1376] [Q&A] [answered]
Hello!
I've read the documentation and assume there is no way to do what I want, but I still want to ask in the case if I am wrong:
With `multi-select` option the user can manually select multiple rows with `Shift+Enter`. Is there any way to call rofi with multiple rows being automatically selected so that the user doesn't need to manually select them? Something like `selected-row` option accepting multiple indices of the rows to be selected? What I need is to manually select multiple rows, perform some `kb-custom-*` action on the selected rows and call rofi again with the previosly selected rows being still selected(like as with `Shift+Enter`). Is it possible?

**Comment by DaveDavenport** [comment, marked as answer]
No, this is currently not possible. Have not thought to hard about it, so a disclaimer here, I think it could be easily added to dmenu mode.

**Reply by Alexey104** [reply]
Ok, thank you for the response!


>I think it could be easily added to dmenu mode.

Would be nice!

Hmm... For some reason the `Mark as answer` button doesn't work for me.

**Reply by DaveDavenport** [reply]
I clicked it, not sure why it didn't work.
